rush.json

```diff
@@ -2,9 +2,6 @@
   "$schema": "https://developer.microsoft.com/json-schemas/rush/v5/rush.schema.json",
   "rushVersion": "5.112.2",
   "pnpmVersion": "7.32.2",
-  "pnpmOptions": {
-    "useWorkspaces": true

```

Comment:
any reason for removing this?

Reply:
Moved to `pnpm-config.json` as requested by rush